lib/humidifier/stack.rb

```diff
@@ -46,8 +47,16 @@ def to_cf
       define_method(method) { |opts = {}| AwsShim.send(method, SdkPayload.new(self, opts)) }
     end
 
+    # Increment the default identifier
+    def self.next_default_identifier
+      @count = (@count || 0) + 1
+      "humidifier-stack-template-#{@count}"
```

Comment:
would basing this on stack.name be easier for users to debug? possibly with a zero-padded date suffix a la autodeploy naming.